src/webui/api/appcontroller.cpp

```diff
@@ -319,6 +319,7 @@ void AppController::preferencesAction()
 
     // RSS settings
     data[u"rss_refresh_interval"_s] = RSS::Session::instance()->refreshInterval();
+    data[u"rss_fetch_delay"_s] = static_cast<int>(RSS::Session::instance()->fetchDelay().count());
```

Comment:
Lets use the integer type that won't ever clip the values:
```suggestion
    data[u"rss_fetch_delay"_s] = static_cast<qlonglong>(RSS::Session::instance()->fetchDelay().count());
```

Reply:
Didn't think about the edge case of users going that high with the delay.
Fixed with https://github.com/qbittorrent/qBittorrent/pull/19801/commits/aa9608600f1584fc77f3ad8d3f194b73f446ea08